app/services/foreman_ansible/playbooks_importer.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+module ForemanAnsible
+  # Imports playbooks from smart proxy
+  class PlaybooksImporter
+    include ::ForemanAnsible::ProxyAPI
+    delegate :playbooks, :playbooks_names, :to => :proxy_api
+
+    def initialize(proxy = nil)
+      @ansible_proxy = proxy
+    end
+
+    def import_playbooks(playbooks_names)
+      import_playbooks_content playbooks(playbooks_names)
+    end
+
+    def import_playbooks_content(playbooks)
+      params = { changed: { update: [], create: [] } }
+      playbooks.each do |playbook|
+        new_playbook = parse_playbook playbook
+        job_template_id = JobTemplate.where(name: new_playbook[:name]).pluck(:id).first
+        old_var_block = PlaybooksImporterHelper.check_vars_block new_playbook[:old_vars_block]
```

Comment:
this naming is quite strange. Why do we have `old_var_block` is there a new one as well? why don't we have just `vars`?